discovery/src/main/java/com/ibm/watson/developer_cloud/discovery/v1/model/NoticeQueryResult.java

```diff
@@ -0,0 +1,243 @@
+/*
+ * Copyright 2017 IBM Corp. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package com.ibm.watson.developer_cloud.discovery.v1.model;
+
+import com.google.gson.annotations.SerializedName;
+import com.ibm.watson.developer_cloud.service.model.GenericModel;
+
+import java.util.Date;
+import java.util.Map;
+
+/**
+ * NoticeQueryResult.
+ */
+public class NoticeQueryResult extends GenericModel {
+
+  /**
+   * Severity level of the notice.
+   */
+  public enum Severity {
+
+    /** warning. */
+    @SerializedName("warning")WARNING,
+
+    /** error. */
+    @SerializedName("error")ERROR
+  }
+
+  /** The unique identifier of the document. */
+  private String id;
+  /** The confidence score of the result's analysis. Scores range from 0 to 1, with a higher score indicating greater
+   *  confidence. */
+  private Double score;
+  /** Status of the processed document which can be 'OK', 'ERROR' depending on if the document type and tags are
+   * accepted by the service. */
+  private String status;
```

Comment:
the reason I ask is I didn't see a `status` in the `NoticeQueryResponse`:

```
NoticeQueryResult {
id (string, optional),
score (number, optional),
notice_id (string, optional): Identifies the notice. Many notices may have the same ID. This field exists so that user applications can programatically identify a notice and take automatic corrective action. ,
created (string, optional),
document_id (string, optional),
severity (string, optional) = ['warning', 'error'],
step (string, optional),
description (string, optional)
}
```